ipykernel/kernelspec.py

```diff
@@ -118,9 +122,13 @@ def install(kernel_spec_manager=None, user=False, kernel_name=KERNEL_NAME, displ
         overrides = dict(display_name=display_name)
     else:
         overrides = None
-    path = write_kernel_spec(overrides=overrides)
-    dest = kernel_spec_manager.install_kernel_spec(path,
-        kernel_name=kernel_name, user=user, prefix=prefix)
+    if profile:
+        extra_arguments = ["--profile", profile]
```

Comment:
Good idea, maybe it would be enough with e.g. "Python X [Y]" to make it more compact?